### PR TITLE
Aug 2018 MOOSEletter

### PR DESCRIPTION
**Pull request by aeslaughter**
@idaholab/moose-team Please add your work.

(refs #11446)

### CODE REVIEW

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# MOOSE Newsletter (August 2018)
+
+## MooseDocs Plots
+
+It is not possible to include line plots in MOOSE markdown. The plots are created using
```
Comment (andrsd) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: not => now (?)

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+[]
+```
+
+
+## Threaded General User Objects
+
+Up until now, general user objects were executed in a single threaded region, even if the
+application ran with multiple-threads.  Now, it is possible to have threaded general user objects.
+Users need to inherit from `ThreadedGeneralUserObject` class and MOOSE will create one threaded
+instance per thread. Then, it is possible to safely access the user object from each threaded. This
```
Comment (jwpeterson) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: typo: "safely access the user object from each threaded." 

Should this be "each thread" or "threaded instance" or ?

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+is not possible in the normal GeneralUserObject unless the access is guarded with a spin mutex.
+
+The `fluid_properties` module was updated to use the `ThreadedGeneralUserObject` to ensure that
+fluid property lookup is always thread safe without the loss of performance.
+
+Note that `ThreadedGeneralUserObject`s are not available when users use pthreads as their threading
+model. The reason is that if a `ThreadedGeneralUserObject` (which runs in a threaded region) created
+a threaded section, there would be `n_threads + n_threads^2` threads active which would yield in a
```
Comment (jwpeterson) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: typo: "yield in a performance loss"

change to: "cause a performance loss"

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+## Threaded General User Objects
+
+Up until now, general user objects were executed in a single threaded region, even if the
+application ran with multiple-threads.  Now, it is possible to have threaded general user objects.
+Users need to inherit from `ThreadedGeneralUserObject` class and MOOSE will create one threaded
+instance per thread. Then, it is possible to safely access the user object from each threaded. This
+is not possible in the normal GeneralUserObject unless the access is guarded with a spin mutex.
+
+The `fluid_properties` module was updated to use the `ThreadedGeneralUserObject` to ensure that
+fluid property lookup is always thread safe without the loss of performance.
```
Comment (jwpeterson) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: I don't understand the sentence. Do you mean "thread safe *and* there is no loss of performance"?

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+a threaded section, there would be `n_threads + n_threads^2` threads active which would yield in a
+performance loss.  If threaded general user objects are needed, they will run properly with OpenMP
+or TBB.
+
+[plotly]: https://plot.ly/
+
+## Working with a different version of libMesh
+
+A new flag has been added to the update_and_rebuild.sh script so that you can work with a different
+version of libmesh. This can be handy when a new feature makes it's way into libMesh but has not
```
Comment (jwpeterson) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: typo: it's -> its

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+been rolled out in MOOSE yet. To use this flag, simply checkout the version of libMesh that you want
```
Comment (jwpeterson) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: typo: "checkout" -> "check out"

File: modules/combined/doc/content/newsletter/2018_08_17.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+and supply "--skip-submodule-update" on the command line to keep that version when you build libMesh.
+
+## GrainTracker Enhancements
+
+For those people working with phase-field and using the GrainTracker, several enhacements and updates
+were merged in during July. First, a significant algorithmic change where merging is performed
```
Comment (jwpeterson) [inline modules/combined/doc/content/newsletter/2018_08_17.md]: We are using the word "merge" to refer to `git merge` and merging of grains, so I will reword this paragraph.

**Review by jwpeterson — COMMENTED** [review]
I found a few small things which I will fix momentarily.

**Review by jwpeterson — APPROVED** [review]

**Comment by aeslaughter** [comment]
@jwpeterson I will send the message when this is merged to master. Thanks for editing.

**Pull request merged**